### PR TITLE
MLPAB-2035 Use lpa-store for registered status

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: internal/lpastore/update.go [hunk 1]
```diff
@@ -36,6 +36,14 @@ func (c *Client) sendUpdate(ctx context.Context, lpaUID string, actorUID actorui
 	return c.do(ctx, actorUID, req, nil)
 }
 
+func (c *Client) SendRegister(ctx context.Context, lpaUID string) error {
```
Comment (hawx) [inline internal/lpastore/update.go:39]: I don't really like this, but the alternative was to make `SendUpdate` or `Do` public, which I think would be worse

File: internal/lpastore/resolving_service.go [hunk 2]
```diff
@@ -61,11 +64,8 @@ func (s *ResolvingService) Get(ctx context.Context) (*Lpa, error) {
 		// copy the relationship as it isn't stored in the lpastore.
 		lpa.CertificateProvider.Relationship = donor.CertificateProvider.Relationship
 
-		// TODO: eventually we'll need to remove the RegisteredAt field as mlpa
-		// won't be tracking that data, then we'll need to figure out how to expose
-		// the data for testing
-		if !donor.RegisteredAt.IsZero() {
-			lpa.RegisteredAt = date.FromTime(donor.RegisteredAt)
+		if lpa.WithdrawnAt.IsZero() {
```
Comment (hawx) [inline internal/lpastore/resolving_service.go:67]: I'm going to add another ticket for this one 😄

File: internal/lpastore/update.go (hunk 1 above)
Comment (acsauk) [reply internal/lpastore/update.go:39]: Fair compromise 👍

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**